### PR TITLE
Contributions page

### PR DESCRIPTION
**Pull request by CheezBallzPi**
Feel free to ask for changes 😛

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,123 @@
...
+
+The blog gives a good explanation on how to manually use commands to use the git flow technique. Although this is only recommended for people experienced in git, you can try it if you want. Just don't delete all our code. 
+
+## How to add code
+
+So let's say you have a great idea for the robot, such as better code for the shooter or something. How would you put this code on GitHub?
+
+DISCLAIMER: Again, this guide assumes that you have prior git knowledge. PLEASE learn git before you attempt this. You WILL break things if you don't.
+
+### 1. Get the code from GitHub
```
Comment (P1n3appl3) [inline CONTRIBUTING.md]: Tell them to configure their ide and give a link to the gradlerio for dummies wiki post

Comment (CheezBallzPi) [inline CONTRIBUTING.md]: *groan* Do I haaaave to? 😑 

Which wiki post?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,123 @@
...
+
+With extension: `git flow feature start <name>`
+
+Without extension: `git checkout -b <name> develop`
+
+### 3. Add your code!
+
+Type up your code in whatever text editor you choose, and commit using:
+
+`git commit -a -m "Title message" -m "Description (optional)"`
```
Comment (P1n3appl3) [inline CONTRIBUTING.md]: Add a reminder to use imperative titles

File: CONTRIBUTING.md (hunk 1 above)
Comment (P1n3appl3) [inline CONTRIBUTING.md]: https://github.com/RoboEagles4828/2017Robot/wiki/GradleRIO-for-dummies

**Comment by CheezBallzPi** [comment]
@P1n3appl3 Have a look

**Pull request merged**